### PR TITLE
Adding in more specific padding and updating h3 variable

### LINKED ISSUE #187: Design System - Badges - ds-label ds-label-outline
Update L, R Padding: 8px, T, B Padding: 4px

Update font-weight: Medium

Update color to Hex value

Thx,

Ismael

### PR DESCRIPTION
**Pull request by fuhton**
Fixes #187 #188 & #212

### CODE REVIEW

File: src/scss/_badges.scss [hunk 1]
```diff
@@ -20,6 +20,21 @@
 	color: $text-primary;
 }
 
+.label-default {
+	background-color: $grey-6;
```
Comment (RachelRVasquez) [inline src/scss/_badges.scss]: @fuhton I submitted my own PR by accident. You can disregard mine but for this area, let's go with my version. If you just overwrite the existing Bootstrap variable - it will automatically set this default color without having to write extra styles.

https://github.com/JDRF/design-system/pull/252

Comment (fuhton) [inline src/scss/_badges.scss]: Why would I be adding yet another variable?

File: src/scss/_badges.scss [hunk 2]
```diff
@@ -20,6 +20,21 @@
 	color: $text-primary;
 }
 
+.label-default {
+	background-color: $label-default-bg;
```
Comment (RachelRVasquez) [inline src/scss/_badges.scss]: @fuhton Now that you've overwritten the variable, Bootstrap will have this style already. This can be removed.

**Comment by RachelRVasquez** [comment]
@fuhton Looks good to me. Merging!
